src/transformers/models/llama/tokenization_llama_fast.py

```diff
@@ -119,8 +121,15 @@ def __init__(
         add_bos_token=True,
         add_eos_token=False,
         use_default_system_prompt=False,
+        add_prefix_space=None,
         **kwargs,
     ):
+        if add_prefix_space is not None:
+            logger.warning_once(
+                "You set `add_prefix_space`. The tokenizer needs to be converted from the slow tokenizers"
+            )
+            kwargs["from_slow"] = True
```

Comment:
How long does it take to convert the tokenizer from slow? If it's quick we can move it to `info`

Reply:
Around 10 seconds I believe!